From: uazo <uazo@users.noreply.github.com>
Date: Tue, 1 Jul 2025 14:50:29 +0000
Subject: Temp disable UseContextSnapshot

Temporarily disable the context snapshot of blink and use that of v8.

License: GPL-2.0-or-later - https://spdx.org/licenses/GPL-2.0-or-later.html
---
 content/public/common/content_features.cc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/content/public/common/content_features.cc b/content/public/common/content_features.cc
--- a/content/public/common/content_features.cc
+++ b/content/public/common/content_features.cc
@@ -1072,6 +1072,7 @@ BASE_FEATURE(kUnrestrictedSharedArrayBuffer, base::FEATURE_DISABLED_BY_DEFAULT);
 #if BUILDFLAG(IS_ANDROID) && BUILDFLAG(INCLUDE_BOTH_V8_SNAPSHOTS)
 // If enabled, blink's context snapshot is used rather than the v8 snapshot.
 BASE_FEATURE(kUseContextSnapshot, base::FEATURE_ENABLED_BY_DEFAULT);
+SET_CROMITE_FEATURE_DISABLED(kUseContextSnapshot);
 #endif
 
 // Enables comparing browser and renderer's DidCommitProvisionalLoadParams in
--
